Makefile

```diff
@@ -321,8 +321,8 @@ endef
 # don't inadvertently link to /lib/libgcc_s.so.1, which is incompatible with
 # libgfortran, and on Windows we copy them in earlier as well.
 ifeq (,$(findstring $(OS),FreeBSD WINNT))
-julia-base: $(build_libdir)/libgfortran.$(SHLIB_EXT)
-$(build_libdir)/libgfortran.$(SHLIB_EXT): | $(build_libdir)
+julia-base: $(build_libdir)/libgfortran*.$(SHLIB_EXT)*
```

Comment:
That’s a weird filename. I don’t think we should be making files with ‘*’ in the name

Reply:
We never make a file with `*` in the name.  The first time `make` is run, there is no file that matches the rule pattern, and so the rule does indeed expand to a strange filename with `*` in it.  But that filename is never used, it just forces the rule to be run.  The second time you run `make`, it expands to match (for example) `libgfortran.5.dylib`, and so then does not run the rule a second time.